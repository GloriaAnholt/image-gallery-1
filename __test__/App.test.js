import React from 'react';
import ReactDOM from 'react-dom';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import ImageGallery from '../src/App';


describe('The Image Gallery React App', () => {

    it('Renders ths app', () => {
        const rendered = shallow(<ImageGallery />);
        expect(toJson(rendered)).toMatchSnapshot();
    });

})