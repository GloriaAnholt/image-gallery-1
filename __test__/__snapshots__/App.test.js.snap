exports[`The Image Gallery React App Renders ths app 1`] = `
<div>
  <SelectorButtons
    setView={[Function]} />
  <SelectView
    view="thumb" />
</div>
`;
